test/integration/acceptance/add_asset_qty_test.cpp

```diff
@@ -54,7 +52,8 @@ TEST_F(AddAssetQuantity, NoPermissions) {
       .skipProposal()
       .checkVerifiedProposal(
           [](auto &proposal) { ASSERT_EQ(proposal->transactions().size(), 0); })
-      .done();
+      .checkBlock(
```

Comment:
Seems redundant, doesn't it?

Reply:
No, it is needed for:
1. We explicitly check that no transaction appeared in a block
2. Currently without that, ITF-tests may got stuck